.github/workflows/run-metrics-latest.yml

```diff
@@ -2,9 +2,8 @@ name: Run Metrics Latest
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - main
+  schedule:
+    - cron: "8 0 * * *" # Runs daily at 8am UTC (1am PST)
```

Comment:
Could you please update the job name in this workflow from `all-integration-tests-latest:` to `run-metrics-with-latest`